interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
It is not proper to call it `consts.js`, because in this module, one service is const and the other is a value. What about `app/utils.js`?